src/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterEnum.cs

```diff
@@ -31,14 +34,7 @@ public override bool TryRead(Type valueType, ref Utf8JsonReader reader, out TVal
                 throw new NotImplementedException(SR.EnumConverterNotImplemented);
```

Comment:
Since you are now using the older `TryParse` which is in netstandard 2.0 this `#if` can be removed.